src/registry/editgrid/preview.tsx

```diff
@@ -43,27 +22,27 @@ const Preview: React.FC<ComponentPreviewProps<EditGridComponentSchema>> = ({comp
       tooltip={tooltip}
     >
       <div style={{display: 'flex', flexDirection: 'column', gap: '1em'}}>
```

Comment:
I am a bit confused..Shouldn't we put all the css in a separate file now that we got rid of `@emotion/css`

Reply:
inline `style` props do not use `@emotion/css` at all - but yeah, they will need to be converted/cleaned up too at some point.

My main goal here is being able to drop the CSS-in-JS dependency and being able to use SASS syntax.